src/thunderbots_msgs/msg/RobotStatus.msg

```diff
@@ -0,0 +1,82 @@
+
+# The rough maximum full-scale deflection of the laser sensor.
+float64 break_beam_scale
```

Comment:
deflection? scale? what do these mean?

Reply:
unfortunately, I have no idea

Reply:
Ah, I believe that this is the estimated maximum value for the break beam reading (the "deflection" part comes from the needle deflection on an older analog reading: https://electronics.stackexchange.com/questions/297210/what-is-full-scale-deflection). Could we move this down with `break_beam_reading` and add a quick comment this effect so it's clear?

Reply:
done :+1: